tests/sdk/test_file_storage.py

```
@@ -0,0 +1,97 @@
+        pytest.raises(ValueError, match="force=True"),
+    ):
+        files_client.download_uploaded_file(file_name=file_path.name, local_dir=str(tmp_path))
+
+    request.assert_not_called()
+
+
+@pytest.mark.parametrize(
+    "file_name",
+    [
+        "../outside.txt",
+        "/tmp/outside.txt",
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/outside.txt"

(S108)